src/features/dotnetTest.ts

```diff
@@ -545,11 +545,18 @@ class DebugEventListener {
         this._fileName = fileName;
         this._server = server;
         this._eventStream = eventStream;
-        // NOTE: The max pipe name on OSX is fairly small, so this name shouldn't bee too long.
-        const pipeSuffix = "T-" + process.pid;
-        if (os.platform() === 'win32') {
+        const pipeSuffix = "TestDebugEvents-" + process.pid;
```

Comment:
I would suggest getting rid of `pipeSuffix` and using something different on non-Windows. Something like:

```
 if (platform === 'win32') {
            this._pipePath = "\\\\.\\pipe\\Microsoft.VSCode.CSharpExt.TestDebugEvents" + process.pid;
        } else {
            // NOTE: The max pipe name on *NIX is fairly small, so this name shouldn't be too long.
            let tmpdir = utils.getUnixTempDirectory();
            this._pipePath = path.join(tmpdir,"vscode-csharp-tde-" + process.pid); // NOTE: The .NET folks use much longer names than this on Linux, so I assume it is okay there, if this is too long on osx, maybe drop the 'vscode-'??
        }
```

Reply:
I opted to use `ms-dotnettools.csharp-tde` as similar is used for . A path like `/tmp/ms-dotnettools.csharp-tde-272809` has 37 chars. According to [this](https://serverfault.com/a/641388), there should be plenty of characters left to satisfy OSX's 104 char limit.

Reply:
To be clear, `$TMPDIR` on OSX (usually? always?) looks like "/var/folders/zz/yyc7kdwj53b927x9n0wngplh0000gq/T/" (49 characters on my mac).

But still, we should be okay. Sounds good to me.